Sources/TensorFlow/Operators/NN.swift

```diff
@@ -12,6 +12,44 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//===------------------------------------------------------------------------------------------===//
+// Core
+//===------------------------------------------------------------------------------------------===//
+
+private extension Tensor where Scalar: TensorFlowFloatingPoint {
+    /// Returns a dropout given a probability and noise shape.
+    @differentiable(wrt: self where Scalar: Differentiable)
+    func droppingOut(probability: Double, noiseShape: TensorShape) -> Tensor {
```

Comment:
```suggestion
    func droppingOut(withProbability probability: Double, noiseShape: TensorShape) -> Tensor {
```

Reply:
“With” is often considered as a meaningless word in argument labels in Swift APIs (though it’s not formally documented). We’ve been avoiding “with” unless it’s both semantically and grammatically significant.